packages/libs/react-components/src/components/IconButton/IconButton.tsx

```diff
@@ -1,20 +1,24 @@
-import { styled } from '../../styles';
+import { config, styled } from '../../styles';
 import { SystemIcons } from '../Icons';
 
+import { PropertyValue } from '@stitches/react';
 import React, { FC } from 'react';
 
 export interface IIConButtonProps
   extends React.HTMLAttributes<HTMLButtonElement> {
   icon: typeof SystemIcons[keyof typeof SystemIcons];
   onClick: React.MouseEventHandler<HTMLButtonElement>;
   title: string;
+  color: React.HTMLAttributes<HTMLButtonElement>['color'] &
+    PropertyValue<'backgroundColor', typeof config>;
 }
```

Comment:
Do you think this would work?

```suggestion
export interface IIConButtonProps
  extends Omit<React.HTMLAttributes<HTMLButtonElement>, "color"> {
  icon: typeof SystemIcons[keyof typeof SystemIcons];
  onClick: React.MouseEventHandler<HTMLButtonElement>;
  title: string;
  color: PropertyValue<'backgroundColor', typeof config>;
}
```